test-suite/fhevm/docker-compose/gateway-node-docker-compose.yml

```diff
@@ -20,3 +20,9 @@ services:
       - "${MNEMONIC}"
     ports:
       - "8546:8546"
+    networks:
+      fhevm_default:
+
+networks:
```

Comment:
Is this network config needed at this point? Also for host-docker-compose, etc. @rudy-6-4